src/chat/commands/ama.ts

```diff
@@ -0,0 +1,24 @@
+import { OnCommandEvent, OnSayEvent } from "../../models"
+import { EventBus, Events } from "../../events"
+import { ShouldThrottle } from '../shouldThrottle'
+
+/**
+ * Sends a message to chat with details about sending stuff to BBB
```

Comment:
You left a bit of Copy/Pasta on the plate. ![bitmoji](https://sdk.bitmoji.com/render/panel/b2541e70-513d-4db6-bb3f-b7f1b3ac92c5-b24761f5-99bf-4885-9465-63838b10c2c3-v1.png?transparent=1&palette=1&width=246)